Makefile

```diff
@@ -145,5 +145,65 @@ snapshot:
 try-runtime-upgrade:
 	RUST_LOG=debug cargo run --bin parallel --features try-runtime -- try-runtime --chain $(CHAIN) --wasm-execution=compiled on-runtime-upgrade snap -s snapshot.bin
 
+.PHONY: local-spec
+local-spec:
+	../polkadot/target/release/polkadot build-spec \
+		 --chain rococo-local \
+		 --raw \
+		 --disable-default-bootnode > ../polkadot/rococo_local.json
+
+.PHONY: local-relay-alice
+local-relay-alice:
+	../polkadot/target/release/polkadot \
+		--chain ../polkadot/rococo_local.json \
+		-d ./resources/cumulus_relay0 \
+		--validator \
+		--alice \
+		--port 50555 \
+		--node-key 0000000000000000000000000000000000000000000000000000000000000001
+
+.PHONY: local-relay-bob
+local-relay-bob:
+	../polkadot/target/release/polkadot \
+		--chain ../polkadot/rococo_local.json \
```

Comment:
can we simplify this a bit since we should launch 4 relaychain nodes using `parallelfinance/polkadot`, maybe we can create a for loop to launch them all at once